recipes/qpdf/all/conanfile.py

```diff
@@ -62,8 +62,9 @@ def layout(self):
 
     def requirements(self):
         # https://qpdf.readthedocs.io/en/stable/installation.html#basic-dependencies
-        self.requires("zlib/1.2.12")
+        self.requires("zlib/1.2.13")
         if self.options.with_ssl == "openssl":
+            # TODO: update to 3.x branch of openssl, when zlib is able to handle it
```

Comment:
Isn't the dependencies the other way around? 😖

Reply:
A yes, your are right. I think it it not necessary to add the comment here after all, since updating dependencies is like an on-going todo... I would just now remove the comment to solve the discussion here.